src/reporting/interfaces/ExtractionProcess.ts

```diff
@@ -0,0 +1,188 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { Link, PagedResponseLinks } from "./Links";
+
+/**
+ * Contains contextual hyperlinks to related data.
+ * @export
+ * @interface ExtractionLinks
+ */
+export interface ExtractionLinks {
+  /**
+   *
+   * @type {Link}
+   * @memberof ExtractionLinks
+   */
+  logs: Link;
+}
+
+/**
+ * Defines a single Extraction Log response.
+ * @export
+ * @interface ExtractionLog
+ */
+export interface ExtractionLog {
+  /**
+   * The state of the Extraction at specific time.
+   * @type {string}
+   * @memberof ExtractionLog
+   */
+  state: ExtractorState;
+  /**
+   * The Reason explaining why state has specific value.
+   * @type {string}
+   * @memberof ExtractionLog
+   */
+  reason: string;
+  /**
+   * Date when Extraction Log entity was created.
+   * @type {string}
+   * @memberof ExtractionLog
+   */
+  dateTime: string;
+  /**
+   * Unique Identifier for the Extraction Run.
+   * @type {string}
+   * @memberof ExtractionLog
+   */
+  jobId: string;
+  /**
+   * The type of the contextId.
+   * @type {string}
+   * @memberof ExtractionLog
+   */
+  contextType: string;
+  /**
+   * The Context Id.
+   * @type {string}
+   * @memberof ExtractionLog
+   */
+  contextId: string;
+  /**
+   * The level of an issue. If Extraction Log entity is not an issue it is marked as \"Information\".
+   * @type {string}
+   * @memberof ExtractionLog
+   */
+  level: string;
+  /**
+   * The issue category. If Extraction Log entity is not an issue this is marked as \"Unknown\".
+   * @type {string}
+   * @memberof ExtractionLog
+   */
+  category: string;
+  /**
+   * Message which specifies why certain issue occured. If Extraction Log entity is not an issue Message is null.
+   * @type {string}
+   * @memberof ExtractionLog
+   */
+  message?: string;
+  /**
+   * If Extraction Log contains an issue, this is marked as true.
+   * @type {boolean}
+   * @memberof ExtractionLog
+   */
+  containsIssues: boolean;
+}
+
+/**
+ * List of Extraction Logs.
+ * @export
+ * @interface ExtractionLogCollection
+ */
+export interface ExtractionLogCollection {
+  /**
+   * List of Extraction Logs.
+   * @type {Array<ExtractionLog>}
+   * @memberof ExtractionLogCollection
+   */
+  logs: Array<ExtractionLog>;
+  /**
+   *
+   * @type {PagedResponseLinks}
+   * @memberof ExtractionLogCollection
+   */
+  _links: PagedResponseLinks;
+}
+
+/**
+ * Metadata associated with a data extraction run.
+ * @export
+ * @interface ExtractionRunSingle
+ */
+export interface ExtractionRunSingle {
```

Comment:
This is how ExtractionRunSingle response can look like:
```
{
    "run": {
        "id": "jg0156461l5abeom8v1mslhlkc-cmar9d9hjof2hu4tidj4rmiur5i6crul",
        "_links": {
            "status": {
                "href": "https://api.bentley.com/insights/reporting/datasources/extraction/status/40f71ed2-31a3-45d3-bce0-5c9f3f112c03"
            }
        }
    }
}
```

The `ExtractionRun` interface is missing `_links`.

Reply:
added _links to extractionRun